src/types/Tab.ts

```diff
@@ -0,0 +1,3 @@
+export interface Tab {
+  id: string, title: string, content: string;
```

Comment:
Properties in the interface should be placed on separate lines for better readability 📖